src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -216,6 +217,30 @@ class certificates extends viewModelBase {
             })
     }
     
+    databasesAccessInfo(certificateDefinition: Raven.Client.ServerWide.Operations.Certificates.CertificateDefinition) {
+        return ko.pureComputed(() => {
+            return certificateModel.resolveDatabasesAccess(certificateDefinition);
+        });
+    }
+    getAccessClass(accessLevel: Raven.Client.ServerWide.Operations.Certificates.DatabaseAccess) {
+        return ko.pureComputed(() => {
+           return accessManager.default.getAccessClass(accessLevel);
+        });
+    }
+
+    getAccessColor(accessLevel: Raven.Client.ServerWide.Operations.Certificates.DatabaseAccess) {
+        return ko.pureComputed(() => {
+            return accessManager.default.getAccessColor(accessLevel);
+        });
+    }
+
+    getAccessInfoText(dbName: string, accessLevel: Raven.Client.ServerWide.Operations.Certificates.DatabaseAccess) {
+        return `<div class="text-left padding padding-xs">
+                   <strong>Name</strong>: ${dbName === "All" ? "All databases" : dbName }<br>
```

Comment:
something is missing here, and I bet you know what's that ;-)

Reply:
you probably mean ```<br/>```
np - fixed - but it does also work w/o

Reply:
no - missing escaping ;-)

Reply:
hmm, I dropped escaping here because I added escaping in method:
`resolveDatabasesAccess` in certificateModel.ts
I can remove from there and add here...

Reply:
hmm I think it isi better place as we format db name here - and we should remove escaping from the other method.

Reply:
so it general we shoulnt' return and operation on raw data and format almost just before showing on UI.

Reply:
done